docs/06-concepts/18-testing/02-the-basics.md

```diff
@@ -0,0 +1,248 @@
+# The basics
+
+## Using `sessionBuilder` to set up a test scenario
```

Comment:
Suggestion: Shorten to `Set up test scenario`.

Menu looks a bit cramped with this long of a headline.